lib/restAPIs/userManagement.js

```diff
@@ -19,49 +18,68 @@ module.exports = class RestTopic extends RestUtil {
     if (abbreviationBegins) {
       queryParameter.abbreviationBegins = abbreviationBegins;
     }
-    return this._makeRequest(this._getBase(), queryParameter, 'GET')
-      .then(result => this.throwIfFailed(result, 200));
+    return this._makeRequest(
+      this._getBase(),
+      queryParameter,
+      'GET'
+    ).then((result) => this.throwIfFailed(result, 200));
```

Comment:
Trim parentheses from result param.

Reply:
fixed. It was caused by Prettifier vs code plugin.